### PR TITLE
Update Packages folder - simplify

### PR DESCRIPTION
**Pull request by ArveVM**
Replacement/simplification of point I was trying to make in #26320. Not sure what happend to that one, I might have cocked up something so it got stuck;)

have tried to simplify;
- removed the option for !include_dir_merge_named from origninal doc, so now this is all about !include_dir_named
- added info on this way of including keep indentation "as if written in configuration.yaml"

## Proposed change

The purpose of this PR is to be able to refer to documentation when urging people to use "packages-folders with the indentation of configuration.yaml" 

Currently I've seen several examples where they have read the documentation and set up with !include_dir_merge_named and having to keep up with different indentation than most examples that can be found in communities etc.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/packages.markdown [hunk 1]
```diff
@@ -74,33 +74,16 @@ Components inside packages can only specify platform entries using configuration
 
 ## Create a packages folder
 
-One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In the packages directory you can store any number of packages in a YAML file. This entry in your `configuration.yaml` will load all packages:
+One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In the packages directory you can store any number of packages in a YAML files. This entry in your `configuration.yaml` will load all YAML-files in Packages-folder and subfolders:
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown]:
```suggestion
One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In the packages directory, you can store any number of packages in a YAML file. This entry in your `configuration.yaml` will load all YAML-files in this _packages_ folder and its subfolders:
```

File: source/_docs/configuration/packages.markdown [hunk 2]
```diff
@@ -74,33 +74,16 @@ Components inside packages can only specify platform entries using configuration
...
 
 ```yaml
 homeassistant:
   packages: !include_dir_named packages
 ```
 
-This uses the concept splitting the configuration and will include all files in a directory with the keys representing the filenames.
-See the documentation about [splitting the configuration](/docs/configuration/splitting_configuration/) for more information about `!include_dir_named` and other include statements that might be helpful. The benefit of this approach is to pull all configurations required to integrate a system, into one file, rather than across several.
+The benefit of this approach is to pull all configurations required to integrate a system, into one file, rather than across several.
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown]:
```suggestion
The benefit of this approach is to pull all configurations required to integrate a system into one file&mdash;rather than keeping them spread across several files.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

**Review by ArveVM — COMMENTED** [review]
Great improvements - thanks :)

File: source/_docs/configuration/packages.markdown [hunk 3]
```diff
@@ -74,33 +74,16 @@ Components inside packages can only specify platform entries using configuration
...
+One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In the packages directory, you can store any number of packages in a YAML file. This entry in your `configuration.yaml` will load all YAML-files in this _packages_ folder and its subfolders:
 
 ```yaml
 homeassistant:
   packages: !include_dir_named packages
 ```
 
-This uses the concept splitting the configuration and will include all files in a directory with the keys representing the filenames.
-See the documentation about [splitting the configuration](/docs/configuration/splitting_configuration/) for more information about `!include_dir_named` and other include statements that might be helpful. The benefit of this approach is to pull all configurations required to integrate a system, into one file, rather than across several.
+The benefit of this approach is to pull all configurations required to integrate a system into one file&mdash;rather than keeping them spread across several files.
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown:84]: I just noticed that the reference to the documentation on configuration splitting somehow got dropped here. Was this intentional?

File: source/_docs/configuration/packages.markdown [hunk 4]
```diff
@@ -74,33 +74,16 @@ Components inside packages can only specify platform entries using configuration
...
+You can use other !include-methods for packages folder, but with this one you get yaml-indentation 'as if the content was written in configuration.yaml'.
 
-The following example allows to have subfolders in the `packages` folder, which could make managing multiple packages easier by grouping:
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown:87]: Remind me, why would this documentation be better off without this example section? Is it incorrect?

File: source/_docs/configuration/packages.markdown (hunk 3 above)
Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:84]: the intention was to simplify.
There is still a reference to splitting configuration in first paragraph of the page - so it applies to everytning Packages - so why repeat in 'create packages folder'-subsection?

File: source/_docs/configuration/packages.markdown (hunk 4 above)
Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:87]: not tecnically incorrect, but for noobs - it is not clear that the docs show two separate ways of doing this - and that both include files in subfolders.
Tried to separate them more clearly in #26320, but that bounced back on "why the two examples" :)

So just trying to simplify,,, I guess the super advanced users see the splitting config-principles and can do it this way,,, or they'll include by functionality - or another way :)

For myself it is much simpler to explain to peers to use "same indentation as in configuration yaml" ,,, which is not what they currently read out of the current documentatjon (have assisted two who have gone for the !include_dir_merge_named packages option because that supposedly is the only one including subfolders :(  -> therefore the request for simplification :)

File: source/_docs/configuration/packages.markdown (hunk 3 above)
Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:84]: should it perhaps be included again,, to compensate for the removal of the !include_dir_merge_named packages - example?

File: source/_docs/configuration/packages.markdown (hunk 4 above)
Comment (c0ffeeca7) [reply source/_docs/configuration/packages.markdown:87]: Yes, ok, I think I'm starting to see your point. Let me rephrase this in my own words to be sure I really understand. 

1. it is not clearly indicated that this example is using the `include_dir_merge_named` method (while the above example was using the `include_dir_named` method). It is easily overlooked by readers that this is a different method.
2. Because this example is listed under the _Create a packages folder_ title, people work with this example because they think this is the way to go if you want to work with subfolders.
3. The thing then doesn't work if they copy paste content from the config - because `include_dir_merge_named` requires a different indentation.
4. All in all, for new users it is easier to start with just one method, the  `include_dir_named` method, because it allows to copy paste from the config file.

Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:87]: Brilliant ! - why was I not able to write it like that in the first place  :)    

Thanks for spelling it out a bit more structured,
(I'm struggeling a bit, not native english nor techy)

So should we proceed with the simplification on just one sample,, or revert to #26320 and explain better the two separate options?
IMHO it is better to keep documentation and example small, and I don't see that the second example have a great audience/usability - so this is the better way? - - but I'm not the expert ;)

Comment (c0ffeeca7) [reply source/_docs/configuration/packages.markdown:87]: could we continue this conversation in a week? I'm on the road and writing on the phone is not my favorite. I'm hesitant to merge because I'm not sure if deleting the example is the best way forward. Maybe we can do some editing.

Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:87]: sure :)
thanks for taking the time to review - and to clarify "my issue"  :)

Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:87]: > could we continue this conversation in a week? I

Got some time for this one now ??

File: source/_docs/configuration/packages.markdown [hunk 5]
```diff
@@ -74,33 +74,16 @@ Integrations inside packages can only specify platform entries using configurati
...
 
 ```yaml
 homeassistant:
   packages: !include_dir_named packages
 ```
 
-This uses the concept splitting the configuration and will include all files in a directory with the keys representing the filenames.
-See the documentation about [splitting the configuration](/docs/configuration/splitting_configuration/) for more information about `!include_dir_named` and other include statements that might be helpful. The benefit of this approach is to pull all configurations required to integrate a system, into one file, rather than across several.
+The benefit of this approach is to pull all configurations required to integrate a system into one file&mdash;rather than keeping them spread across several files.
+You can use other !include-methods for packages folder, but with this one you get yaml-indentation 'as if the content was written in configuration.yaml'.
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown]:
```suggestion
You can use other `!include` methods for packages; for example `!include_dir_merge_named`. However, unlike `!include_dir_merge_named`, the `!include_dir_named` method uses the same indentation as the configuration.yaml'. This means that you can copy and paste elements from the config file. With `!include_dir_named`, the file name is used as the package name. File names must be unique.
```

File: source/_docs/configuration/packages.markdown [hunk 6]
```diff
@@ -74,33 +74,16 @@ Integrations inside packages can only specify platform entries using configurati
...
 
-The following example allows to have subfolders in the `packages` folder, which could make managing multiple packages easier by grouping:
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown:87]:
```suggestion
Another approach is to organize packages using the `!include_dir_merge_named` method.
This method allows you to have subfolders in the `packages` folder, which could make managing multiple packages easier by grouping:
```
How about leaving the example in, but adding a bit more info before the example?

Comment (c0ffeeca7) [reply source/_docs/configuration/packages.markdown:87]:
```suggestion
With the `!include_dir_merge_named` method, the package name has to be included in the file. The configuration below then needs to be indented accordingly - which means you cannot directly copy and paste from the configuration file.
```
And then, add this information after the example?

File: source/_docs/configuration/packages.markdown [hunk 7]
```diff
@@ -74,33 +74,16 @@ Integrations inside packages can only specify platform entries using configurati
...
-and in `packages/subsystem1/functionality1.yaml`:
-
-```yaml
-subsystem1_functionality1:
-  input_boolean:
-  ...
-  binary_sensor:
-  ...
-  automation:
-```
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown:86]:
```suggestion
With the `!include_dir_merge_named` method, the package name has to be included in the file. The configuration below then needs to be indented accordingly. This means you cannot directly copy and paste from the configuration file.
```
And then, after the example, add the information that you need to be careful when copy-pasting?

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Yes. Thank you for sticking around. 
I've reworked the text a bit. I'm hoping to preserve the second example while pointing out the differences between the two methods. 
What do you think? Would this be a way to move forward?

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ArveVM :+1:
